ds.go

```diff
@@ -107,7 +104,7 @@ func (slice ByName) Less(i, j int) bool {
 }
 
 // Setup create the table which stores credentials
-func Setup() (err error) {
+func Setup(Table *string) (err error) {
```

Comment:
Fair point, did that in main.go after noticing it collided with the table variable used for the TableWriter.